test/functionalTest/cases/2015_notification_templates/notification_templates_reread_cache_from_mongo.test.DISABLED

```diff
@@ -1,46 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
-#
-# This file is part of Orion Context Broker.
-#
-# Orion Context Broker is free software: you can redistribute it and/or
-# modify it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# Orion Context Broker is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
-# General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
-#
-# For those usages not covered by this license please contact with
-# iot_support at tid dot es
-
-# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
-
---NAME--
-Notification Templates, getting the cache from mongo at startup
-
---SHELL-INIT--
-dbInit CB
-brokerStart CB
-accumulatorStart --pretty-print
-
---SHELL--
-
```

Comment:
Yes, there is already a test that populates the sub-cache, then waits for the sub-cache to be refreshed and after that creates an entity to trigger notifications.
Manually I have verified that the sub.cache is refreshed (as you know, refreshing the cache is throwing it away and then repopulating it from mongo - almost identical to what wanted to do in this test).
See notification_templates_cache_refresh.test